tests/conftest.py

```diff
@@ -111,16 +111,18 @@ def xmatch_correct_3n_2t_no_margin(small_sky_xmatch_dir):
 def xmatch_mock(small_sky_xmatch_dir):
     return pd.read_csv(os.path.join(small_sky_xmatch_dir, XMATCH_MOCK_FILE))
 
-
-def assert_divisions_are_correct(catalog):
-    # Check that number of divisions == number of pixels + 1
-    hp_pixels = catalog.get_ordered_healpix_pixels()
-    assert len(catalog._ddf.divisions) == len(hp_pixels) + 1
-    # Check that the divisions are not None
-    assert None not in catalog._ddf.divisions
-    # Check that divisions belong to the correct pixel
-    for division, hp_pixel in zip(catalog._ddf.divisions, hp_pixels):
-        div_pixel = hipscat_id_to_healpix([division], target_order=hp_pixel.order)
-        assert hp_pixel.pixel == div_pixel
-    # The last division corresponds to the HIPSCAT_ID_MAX
-    assert catalog._ddf.divisions[-1] == HIPSCAT_ID_MAX
+@pytest.fixture
+def assert_divisions_are_correct():
```

Comment:
=[

I'm sorry you had to do this.

Reply:
I know right! It works but the linter was complaining about it at a certain point :((